### PR TITLE
Added table when contributor is expanded

### PR DESCRIPTION
**Pull request by bandjeo**

### CODE REVIEW

File: oss-cloud-frontend/src/components/ContributionsPage.vue [hunk 1]
```diff
@@ -17,13 +17,29 @@
       </template>
 
       <template v-slot:item.actions="{ item }">
-          <v-item-group row>
-          <v-btn fab dark color="green" height="30" width="30" class="mx-1" @click="updateStatus(item, 'Visible')"
-            v-if="item.status != 'Visible'">
+        <v-item-group row>
+          <v-btn
```
Comment (banecogic) [inline oss-cloud-frontend/src/components/ContributionsPage.vue]: Let's try to be consistent when defining button element (and other types too).
There are changes where every attribute is in separate column, but then there are others that put everything in one line -> [link](https://github.com/levinine/oss-cloud/pull/19/files#diff-1e0cbd97437ffcf8717aaa61f0a38acbR38) [link](https://github.com/levinine/oss-cloud/pull/19/files#diff-c16288988fe45317499ed65c9bf08c91R9)

I would suggest to put all attributes in one line, eventually events or flow attributes in separate lines.

File: oss-cloud-backend/serverless.yml [hunk 2]
```diff
@@ -47,4 +47,11 @@ functions:
       - http:
           path: contributionStatus
           method: post
+
+  contributorVisibleContributions:
```
Comment (banecogic) [inline oss-cloud-backend/serverless.yml]: Can we merge this into `getContributions` lambda and have only one parametrized API to get these.

P.S. should be get, not post. [link](https://www.w3schools.com/tags/ref_httpmethods.asp)

File: oss-cloud-frontend/src/components/ContributorsPage.vue [hunk 3]
```diff
@@ -1,6 +1,8 @@
 <template>
   <div>
-    <ContributorsTable></ContributorsTable>
+    <v-container fill-heigh>
```
Comment (mjovanovic) [inline oss-cloud-frontend/src/components/ContributorsPage.vue]: typo? 
fill-heigh**t**

Comment (bandjeo) [reply oss-cloud-frontend/src/components/ContributorsPage.vue]: Yes, thank you.

File: oss-cloud-backend/serverless.yml (hunk 2 above)
Comment (bandjeo) [reply oss-cloud-backend/serverless.yml]: Will be merged once search is implemented.

File: oss-cloud-frontend/src/components/ContributionsPage.vue (hunk 1 above)
Comment (bandjeo) [reply oss-cloud-frontend/src/components/ContributionsPage.vue]: Tried three different formatters, all do the same.

**Pull request merged**